src/Input/Input.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import classNames from "classnames";
+import styled from "styled-components";
+
+const InputContainer = styled.div`
+  display: flex;
+`;
+const PrefixIcon = styled.span`
+  background: white;
+
+  border-right: 1px solid grey;
+`;
+const SuffixIcon = styled.span`
+  background: white;
+
+  border-left: 1px solid grey;
+`;
+
+interface InputProps {
+  name: string;
+  type?: string;
```

Comment:
```ts
type InputType =
  | "checkbox"
  | "button"
  | "color"
  | "date"
  | "datetime-local"
  | "email"
  | "file"
  | "hidden"
  | "image"
  | "month"
  | "number"
  | "password"
  | "radio"
  | "range"
  | "reset"
  | "search"
  | "submit"
  | "tel"
  | "text"
  | "time"
  | "url"
  | "week";

interface InputProps {
  name: string;
  type?: InputType;
	// ....
```

Reply:
Most of these will be a separate component, probably we only need to expect these `text, email, password, number, url` types. 

I'll make the change when designs are done.

Reply:
Okay then 👍 I've discovered we had bunch of `type=string` props, which was not valid. So this is quite useful imo.